src/shared/components/mutationMapper/MutationMapperStore.ts

```diff
@@ -52,8 +44,6 @@ export interface IMutationMapperStoreConfig {
 
 export default class MutationMapperStore extends DefaultMutationMapperStore
 {
-    @observable public activeTranscript: string | undefined = undefined;
```

Comment:
Most of the logic implemented here is now move to the default mutation mapper store in the `react-mutation-mapper`